main.py

```diff
@@ -47,7 +46,7 @@ def process_year(self, year):
         print(f"Humidity: {most_humid_day}% on {most_humid_day_date}")
 
 
-    def process_month(self, year, month):
+    def process_monthly_weather(self, year, month):
```

Comment:
This method is extracting average temperature for the month, incorporate that in name as well.